services/graph/pkg/config/application.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+// Application defines the available graph application configuration.
+type Application struct {
+	ID          string `yaml:"id" env:"GRAPH_APPLICATION_ID" desc:"The ocis application id shown in the graph. All app roles are tied to this."`
```

Comment:
@butonic could you add a changelog item here? Just pulled the latest oCIS docker image and had it fail with an error until I added the `GRAPH_APPLICATION_ID` ;)

Reply:
AKA "I went to the changelog first, didn't find anything obvious and ended up here in your PR"

Reply:
sorry, yes ... I hope you got at least a meaningful error message?

Unfortunately, rolling a random application id also is also suboptimal because we would have to write it to the yaml config, which might be readonly ... I'm thinking about a solution for that. maybe store it in the metadata instead of the yaml file ...